### PR TITLE
[Feat] Optimism Deployment

### PR DESCRIPTION
**Pull request by Hephyrius**

### CODE REVIEW

File: contracts/sidechain/README.md [hunk 1]
```diff
@@ -157,6 +158,33 @@ yarn task sidechhain:test:lock-aura --wait <WAIT_N_BLOCKS> --amount <AMOUNT_TO_L
 | View                     | [0x0a6bcB3a0C03aB2Bc8A058ee02ed11D50b494083](https://arbiscan.io/address/0x0a6bcB3a0C03aB2Bc8A058ee02ed11D50b494083) |
 | KeeperMulticall3         | [0x5C97f09506d60B90a817EB547ea4F03Ae990E798](https://arbiscan.io/address/0x5C97f09506d60B90a817EB547ea4F03Ae990E798) |
 
+# <<<<<<< HEAD
```
Comment (0xahtle7) [inline contracts/sidechain/README.md:161]: There were som errors while merging?

File: contracts/sidechain/README.md [hunk 2]
```diff
@@ -157,6 +158,33 @@ yarn task sidechhain:test:lock-aura --wait <WAIT_N_BLOCKS> --amount <AMOUNT_TO_L
...
+| BoosterOwner             | [0xF044eE152C7D731825280350D876CF760181D96F](https://optimistic.etherscan.io/address/0xF044eE152C7D731825280350D876CF760181D96F) |
+| ProtocolDAO              | [0xD86CEB76e9430D3bDE90ded79c82Ae62bc66d68b](https://optimistic.etherscan.io/address/0xD86CEB76e9430D3bDE90ded79c82Ae62bc66d68b) |
+| ClaimZap                 | [0x617963D46B882ecE880Ab18Bc232f513E91FDd47](https://optimistic.etherscan.io/address/0x617963D46B882ecE880Ab18Bc232f513E91FDd47) |
+| Treasury                 | [0x57ACb721FcF3d900B480A90A55191CF8F37ad478](https://optimistic.etherscan.io/address/0x57ACb721FcF3d900B480A90A55191CF8F37ad478) |
+| RewardPoolDepositWrapper | [](https://optimistic.etherscan.io/address/)                                                                                     |
+| BridgeSender             | [0x0451255563e2aca170b2552111837572e7a0bacd](https://optimistic.etherscan.io/address/0x0451255563e2aca170b2552111837572e7a0bacd) |
+| View                     | [](https://optimistic.etherscan.io/address/)                                                                                     |
+
+> > > > > > > a7c3a47 (feat: readme addresses)
```
Comment (0xahtle7) [inline contracts/sidechain/README.md:186]: Merging isues

File: contracts/sidechain/README.md (hunk 1 above)
Comment (0xahtle7) [inline contracts/sidechain/README.md:161]:
```suggestion
```

**Review by 0xahtle7 — APPROVED** [review]
LGTM

**Review by phijfry — APPROVED** [review]

**Pull request merged**